types/src/access_path.rs

```diff
@@ -208,8 +225,8 @@ impl TryFrom<StateKey> for AccessPath {
     type Error = Error;
 
     fn try_from(state_key: StateKey) -> Result<Self> {
-        match state_key.into_inner() {
-            StateKeyInner::AccessPath(access_path) => Ok(access_path),
+        match state_key.inner() {
+            StateKeyInner::AccessPath(access_path) => Ok(access_path.clone()),
```

Comment:
i think we can avoid the clone because state_key is owned, or did you mean to change impl to `TryFrom<&StateKey>`?

Reply:
It doesn't matter, this is for test only. But the state key will be cached Singleton so a clone here is not avoidable. See the stacked (draft) pr.